packages/rest-client/src/main/RestClient.ts

```diff
@@ -98,7 +98,8 @@ export default abstract class RestClient {
    * @param request - The request options including path, query, headers, responseType, and raw flag.
    * @param authOptions - (Optional) Either an AuthOptions object, a raw JWT string, or undefined for no auth.
    * @returns The response body or the full response if raw is true.
-   * @throws Sanitised error if the request fails.
+   * @throws SanitisedError<ErrorData> if the request fails. Note that this is the default behaviour, but can be
```

Comment:
my JSDoc experience is a bit limited but i think if you want to specify the type in an annotation like this, you wrap it in braces
```typescript
/**
 * @throws {SanitisedError<ErrorData>} if the request fails…
 */
```